### ISSUE TITLE
TensorFlow -> PyTorch Model Conversion List

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#2863]
We've made a call to migrate our existing TensorFlow models in PyTorch for better longterm sustainability. Here is a list of TensorFlow models that have yet to be migrated

- [ ] `TensorflowMultitaskIRVClassifier`
- [ ] `RobustMultitaskClassifier`
- [ ] `RobustMultitaskRegressor`
- [ ] `ProgressiveMultitaskClassifier`
- [ ] `ProgressiveMultitaskRegressor`
- [ ] `WeaveModel`
- [ ] `DTNNModel`
- [ ] `DAGModel`
- [ ] `GraphConvModel`
- [x] `MPNNModel`
- [ ] `BasicMolGANModel`
- [ ] `ScScoreModel`
- [ ] `SeqToSeq`
- [ ] `GAN`, `WGAN`
- [ ] `CNN`
- [ ] `TextCNNModel`
- [ ] `AtomicConvModel`
- [ ] `Smiles2Vec`
- [ ] `AtomicConvModel`
- [ ] `ChemCeption`
- [ ] `NormalizingFlowModel`

When doing a conversion, I recommend first implementing the new model in `TorchModel` and writing tests to make sure that the qualitative and quantitative behavior of the new model matches the TensorFlow base implementation. We will migrate models to replace the old implementation over a release or two

**Comment by arunppsg** [comment]
MPNNModel has been migrated to pytorch earlier([ref](https://github.com/deepchem/deepchem/blob/master/deepchem/models/torch_models/mpnn.py)).

**Comment by BalajiAI** [comment]
I would like to take this up.
Any feedback would be valuable to me.

**Comment by BalajiAI** [comment]
Hey @rbharath, GAN, WGAN models are aldready ported to PyTorch.
See here: https://github.com/deepchem/deepchem/pull/3013
But the problem is the PR is in draft.

**Comment by davidRFB** [comment]
Hi @rbharath ! I would like to contribute the RobustMultitask Regressor/Classifiers. However, already saw some Multitask in the pytorch models. If I understand correctly  this add an extra bypass layer, which make it different to the pytorch model. Is there still the help wanted to convert this model ?

**Comment by naseehasalam** [comment]
Working on porting RobustMultitask Regressor and Classifier.

**Comment by gauthamk02** [comment]
Hey, Im working on `ProgressiveMultitaskClassifier` and `ProgressiveMultitaskRegressor`

**Comment by Shiva-sankaran** [comment]
Hi @rbharath , I think it would be great if you can update the list of models, as I see GAN has already been implemented in torch.

**Comment by Shiva-sankaran** [comment]
I am working on TextCNNModel.

**Comment by abuelnasr0** [comment]
Hi @rbharath, I am Mohamed Abu El-Nasr.
I would like to help porting `GraphConvModel` into pytorch, I am working on it, and I will open a PR soon.

**Comment by rbharath** [comment]
@abuelnasr0 Please come by office hours first! This is a particularly challenging project that you should discuss with me first https://forum.deepchem.io/t/announcing-the-deepchem-office-hours/293

**Comment by HridayM25** [comment]
Hi @rbharath, 2 questions : 
1) Is the ScScoreModel being worked on by someone currently? I had seen a PR on the [same.](https://github.com/deepchem/deepchem/pull/3692) 
2) Could I work on the PINN Model if no one else is working on it?
Thanks!

**Comment by Om-Doiphode** [comment]
Hi @rbharath, I am Om Doiphode.
Could I work on porting `RobustMultitaskClassifier` into PyTorch?
Thanks!

**Comment by progs2002** [comment]
> Hey @rbharath ,  I have started working on porting - the `smiles2vec`model. Will open a PR soon.

**Comment by progs2002** [comment]
Hi @rbharath , started working on porting the `Smiles2Vec` model. Will open a PR ASAP.

**Comment by Ibrahimsuf** [comment]
Hi, @rbharath  Could I work on converting ChemCeption to PyTorch?

**Comment by rajeshgayathri2003** [comment]
Hi @rbharath ! Looks like ChemCeption is still open, and I would like to work on the same. I would be happy to discuss the same with you in an office hour.
